limacharlie/organization.go

```diff
@@ -207,6 +207,30 @@ func (o *Organization) CreateOrganization(location, name string, template ...int
 	return resp, nil
 }
 
+func (o *Organization) GetDeleteConfirmationToken() (string, error) {
+	resp := struct {
+		ConfirmationToken string `json:"confirmation,omitempty"`
+	}{}
+	request := makeDefaultRequest(&resp)
+	if err := o.client.reliableRequest(http.MethodGet, fmt.Sprintf("orgs/%s/quota", o.client.options.OID), request); err != nil {
```

Comment:
I think you want to hit the `/delete` API.
```suggestion
	if err := o.client.reliableRequest(http.MethodGet, fmt.Sprintf("orgs/%s/delete", o.client.options.OID), request); err != nil {
```